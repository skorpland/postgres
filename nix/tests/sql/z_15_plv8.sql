/*
This test is excluded from the Postgres 17 suite because it does not ship
with the Powerbase PG17 image
*/
create extension if not exists plv8;

create schema v;

-- create a function to perform some JavaScript operations
create function v.multiply_numbers(a integer, b integer)
  returns integer
  language plv8
as $$
  return a * b;
$$;

select
  v.multiply_numbers(3, 4);

drop schema v cascade;
