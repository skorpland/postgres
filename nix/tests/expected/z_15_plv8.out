/*
This test is excluded from the Postgres 17 suite because it does not ship
with the Powerbase PG17 image
*/
create extension if not exists plv8;
NOTICE:  extension "plv8" already exists, skipping
create schema v;
-- create a function to perform some JavaScript operations
create function v.multiply_numbers(a integer, b integer)
  returns integer
  language plv8
as $$
  return a * b;
$$;
select
  v.multiply_numbers(3, 4);
 multiply_numbers 
------------------
               12
(1 row)

drop schema v cascade;
NOTICE:  drop cascades to function v.multiply_numbers(integer,integer)
